pennylane/workflow/qnode.py

```diff
@@ -292,6 +292,9 @@ class QNode:
         gradient_kwargs (dict): A dictionary of keyword arguments that are passed to the differentiation
             method. Please refer to the :mod:`qml.gradients <.gradients>` module for details
             on supported options for your chosen gradient transform.
+        autograph (bool): Whether to use AutoGraph to convert Python control flow to native PennyLane

```

Comment:
```suggestion
        autograph (bool): *EXPERIMENTAL*  Only applicable when the experimental capture mode is enabled.
        Whether to use AutoGraph to convert Python control flow to native PennyLane
```
Or something like that. We might even want to consider leaving it out of the docstring for the time being.

Reply:
@isaacdevlugt, thoughts?

Reply:
```suggestion
        autograph (bool): *Only applicable when the experimental capture mode is enabled.* Whether to use AutoGraph to convert Python control flow to native PennyLane

```